src/main/fc/cli.c

```diff
@@ -1016,23 +1002,6 @@ static void cliMotorMix(char *cmdline)
         for (uint32_t i = 0; i < MAX_SUPPORTED_MOTORS; i++) {
             customMotorMixerMutable(i)->throttle = 0.0f;
         }
-    } else if (sl_strncasecmp(cmdline, "load", 4) == 0) {
```

Comment:
I'd still keep some common presets inside the firmware. What do you think?

Reply:
I think you are right about restore @stronnag . I will address this issue in a next few commits.
@stronnag modified configurator fully supports previous firmware versions. But problem of backward compatibility is not that big as you might imagine. 

On the other hand, I'm not affarid about 3rd party tools. New firmware still reports mixer (always QuadX) and other things. What you can not do is to setup platform. I doubt users really use android tool to do full setup.

But `preset` setting will have to be present in CLI after all. And Configurator will also display more messages / aids.